addons/hr_timesheet/models/project_project.py

```diff
@@ -18,6 +18,7 @@ class Project(models.Model):
             ('partner_id', '=?', partner_id),
         ]"""
     )
+    analytic_account_active = fields.Boolean(related="analytic_account_id.active")
```

Comment:
```suggestion
    analytic_account_active = fields.Boolean(related="analytic_account_id.active", export_string_translation=False)
```
To avoid exporting the field label to our translation tools, since it's not being displayed anywhere